src/adapters/postgres/user-adapter.ts

```
@@ -1028,9 +1028,12 @@ export class PostgresUserService implements IServicelocator {
     request: any,
     userCreateDto: UserCreateDto,
     academicYearId: string,
-    response: Response,
+    response: Response
   ) {
+    console.log('Create User start logs');
```

ESLint

[error] Replace `'Create·User·start·logs'` with `"Create·User·start·logs"`

(prettier/prettier)